filters.cpp

```diff
@@ -73,17 +74,21 @@ void Filter::flush() {
 
 void Filter::dumpParams() {
```

Comment:
Remove the ` setSerial` method - it makes little sense for this library to carry a pointer to a HardwareSerial instance. If anything, keep the HardwareSerial pointer as an optional argument to the `dumpParams` function, e.g., ` dumpParams(HardwareSerial * serial = Serial)` . The function `dumpParams`  is only supposed to be a debug function for my, and will be likely stripped from the library in future releases.